test/test_2_load_data.py

```diff
@@ -36,7 +36,6 @@ def test_load_dataset(task_name, dataset_name, test, tempdir, image):
     dataset = getattr(task.datasets, dataset_name)
     adata = dataset(test=test)
     utils.asserts.assert_finite(adata.X)
-    utils.asserts.assert_array_equal(adata.X, adata.layers["counts"])
```

Comment:
Maybe we can put a flag on the task? like `task.DEFAULT_LAYER = 'counts'` for all tasks except these, and then assert `adata.X == adata.layers[DEFAULT_LAYER]`?

Reply:
I tried extending the tests to adapt for that. Does this look ok?